docs/templates/plugin.rst.j2

```diff
@@ -135,7 +149,7 @@ Parameters
                     {% endif %}
                     {# Show possible choices and highlight details #}
                     {% if value.choices %}
-                        <ul><b>Choices:</b>
+                        <ul style="margin: 0; padding: 0"><b>Choices:</b>
```

Comment:
What does this achieve?

Reply:
First of all, this should all go into CSS, but you know how difficult it is to ensure this turns up in the right place. And before we were told that CSS was off limits, that's why everything is hard-coded here. Should be fixed at some point though.

Reply:
But it causes a listing to not have trailing whitespace (and so we get more condensed tables).